pkg/contextkeys/contextkeys_test.go

```diff
@@ -0,0 +1,42 @@
+//go:build unit
+// +build unit
+
+package contextkeys
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
```

Comment:
Should add a test for using the contextKey outside of the package (not sure why someone might do that, but it's available)

```
x := contextKey{}
x.SetAccountID(context.Background(), "blah") // This might panic?
```